ui/components/app/token-detected/token-address/token-address.scss

```diff
@@ -0,0 +1,5 @@
+.token-address {
+  & &__copy-link {
```

Comment:
Not sure we should use this syntax. Is it because of specificity issues? I have an PR open to [swap the order of stylesheets](https://github.com/MetaMask/metamask-extension/pull/13745) but it needs thorough testing and I haven't had time to go through it all yet.  https://css-tricks.com/the-sass-ampersand/#aa-wacky-but-working-example-1

<img width="889" alt="Screen Shot 2022-03-21 at 8 39 53 PM" src="https://user-images.githubusercontent.com/8112138/159514918-c3091e87-ab91-4655-8ae0-652e06ac7e77.png">